hacspec/tests/test_util.rs

```diff
@@ -0,0 +1,42 @@
+#![allow(dead_code)]
+
+use hacspec::prelude::*;
+
+pub(crate) fn get_expected(op: &'static str, a: &String, b: &String) -> String {
+    let expected = std::process::Command::new("python")
```

Comment:
Ha ha that's a funny way to define what's "expected" but I buy it :)

Reply:
:laughing: I toyed around with a couple different ways of getting proper math testing in. This seemed the least hassle and "just works".